README.md

```diff
@@ -32,7 +30,7 @@ The following steps will create a Python project generated with the [`tomtom-int
     cookiecutter gh:tomtom-international/cookiecutter-python
     ```
 
-1. Create a new repository on GitHub
+2. Create a new repository on GitHub
```

Comment:
In markdown you don't need to use proper numbering. If you use always 1. it will automatically generate the upcoming numbers. This ensures that you don't need to modify the numbering in case you inject a new point in between